examples/dune

```diff
@@ -15,7 +15,8 @@
   (modules testpatterns colors almondbread crops cursor cuts emoji inline keys
            letters mouse thisbig runes sierpinski rain sierpinski_lwt linear
            life)
-  (libraries common common_lwt))
+  (libraries common common_lwt)
+  (flags -w -32))
```

Comment:
May be take the opportunity to delete this warning :+1:

Reply:
There's value in keeping the unused values in the examples. It is a bit ugly to disable the warning globally, so I've just disabled it on offending values.